venv/.Python

```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python/2.7.14_3/Frameworks/Python.framework/Versions/2.7/Python
```

Comment:
Add `venv/` to .gitignore file so that this won't be committed to your PR.

Reply:
done

Reply:
Still here?